specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/storage.json

```diff
@@ -1868,13 +1871,42 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$maxpagesize",
+            "in": "query",
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 5000,
+            "format": "int32",
+            "description": "Optional, specifies the maximum number of local users that will be included in the list response."
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "type": "string",
+            "description": "Optional. When specified, only local user names starting with the filter will be listed."
+          },
+          {
+            "name": "$include",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "nfsv3"
+            ],
+            "x-ms-enum": {
+              "name": "ListLocalUserIncludeParam",
+              "modelAsString": true
+            },
+            "description": "Optional, when specified, will list local users enabled for the specific protocol. Lists all users by default."
           }
         ],
         "responses": {
           "200": {
             "description": "OK -- List local users successfully.",
             "schema": {
-              "$ref": "#/definitions/LocalUsers"
+              "$ref": "#/definitions/LocalUserListResult"
```

Comment:
This will cause SDK breaking change. Why not add `nextLink` property to the previous `LocalUsers`?